app/lib/methods/subscriptions/room.ts

```diff
@@ -138,6 +138,23 @@ export default class RoomSubscription {
 					reduxStore.dispatch(removeUserTyping(name));
 				}
 			}
+		} else if (ev === 'user-activity') {
+			const { user } = reduxStore.getState().login;
+			const { UI_Use_Real_Name } = reduxStore.getState().settings;
+			const { subscribedRoom } = reduxStore.getState().room;
+			if (subscribedRoom !== _rid) {
+				return;
+			}
+			const [name, activity] = ddpMessage.fields.args;
```

Comment:
considering that activity is an array of activities, what do you think about `activities`?